pkgs/by-name/ta/tacent/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  ninja,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tacent";
+  version = "0.8.18";
+
+  src = fetchFromGitHub {
+    owner = "bluescan";
+    repo = "tacent";
+    tag = finalAttrs.version;
```

Comment:
This should be:
```nix
tag = "v${finalAttrs.version}";
```
Currently this very much has failing builds all over hydra because of this.